packages/icon/src/LionIcon.js

```diff
@@ -111,8 +111,24 @@ export class LionIcon extends LitElement {
     }
   }
 
-  _renderSvg(svgOrModule) {
-    const svg = svgOrModule && svgOrModule.default ? svgOrModule.default : svgOrModule;
-    this.innerHTML = svg;
+  _renderSvg(svg) {
+    this.__validateSvg(svg);
+    render(svg, this);
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  __unwrapSvg(wrappedSvg) {
```

Comment:
maybe make this and the helper below static?

Reply:
ok